source/scripting/World.cpp

```diff
@@ -1024,17 +1024,17 @@ namespace GTA
 	{
 		Native::Function::Call(Native::Hash::ADD_EXPLOSION, position.X, position.Y, position.Z, static_cast<int>(type), radius, true, false, cameraShake);
 	}
-	void World::AddExplosion(Math::Vector3 position, ExplosionType type, float radius, float cameraShake, bool Invis, bool Aubidble)
+	void World::AddExplosion(Math::Vector3 position, ExplosionType type, float radius, float cameraShake, bool Aubidble, bool Invis)
```

Comment:
Of course it would break something. If a script was expecting those parameters to be in the previous order and now they suddenly aren't, then that script would behave different (it would still work, but not do what it was supposed to do). Either way, it doesn't matter in this case.